src/components/MenuPanel/OrganizationSwitcher/FavoriteRow.js

```diff
@@ -25,26 +26,27 @@ class FavoriteRow extends React.Component {
         css={`
           display: flex;
           align-items: center;
+          width: 100%;
```

Comment:
@bpierre I've tried adding `overflow: hidden` here but can't seem to get it to work with the flex declaration.

If a org name is too long this currently happens:

<img width="484" alt="Screen Shot 2019-08-20 at 7 14 12 PM" src="https://user-images.githubusercontent.com/4166642/63391243-170f8980-c3b2-11e9-8d48-3e9509ba58d8.png">

Reply:
I think I fixed that here (the menu is separated so we can reuse it in the onboarding): https://github.com/aragon/aragon/pull/921/commits/2cab20181f8fc12ce42da21c3f5bb52d3dbe42fa

Otherwise the trick is to use `display: block` (even as a flex child) to get `text-overflow: ellipsis` + `overflow: hidden` to work.